crux-lucene/src/crux/lucene.clj

```diff
@@ -238,7 +240,7 @@
                         (.add (StringField. field-crux-eid, (->hash-str e), Field$Store/NO))
```

Comment:
move `(->hash-str e)` up in the `doseq`, above the `[a v]` binding

Reply:
consider re-using Fields here, even just within a Crux document

Reply:
also consider bringing `(keyword->k a)` out to the batch à la `->content-idx-kvs`